bin/tflocal

```diff
@@ -104,6 +104,48 @@ def create_provider_config_file(provider_aliases=None):
     }
     # service names to be excluded (not yet available in TF)
     service_excludes = ["meteringmarketplace"]
+    service_alias_pairs = [
```

Comment:
Can we maybe add a short comment here, and also a pointer to the [docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/custom-service-endpoints#appautoscaling), for future reference? 

Maybe we could also elevate this variable as `SERVICE_ALIASES` to the top level (root scope) of the file, to indicate that this is a configuration/constant that needs to get updated over time.

Reply:
If we elevate this into a global scope variable then we should move the exclusions and replacements there too to keep it consistent.
- service_excludes -> `SERVICE_EXCLUSIONS`
- service_replaces -> `SERVICE_REPLACEMENTS`